cypress/e2e/servers_page.cy.ts

```diff
@@ -20,7 +20,7 @@ describe('Server listing', () => {
     E2EEvent.isVisible(`[id="servers_table"]`);
 
     // count lines
-    E2EEvent.countElements(`[id="servers_table"]`, `[id="server_table_data_line"]`,25);
```

Comment:
this needed to be updated in the keeper repo .. done at https://github.com/SpareCores/sc-keeper/commit/c1cfb2ef2686cd0acd2bbebd374216584be0c008

Reply:
@Palabola I think we might need to rerun the SDK generator? can you pls help?